book/website/communication/binder/zero-to-binder.md

```diff
@@ -205,9 +205,9 @@ The truth is that you'd never do it manually, the built-in package manager `Pkg`
 
 ````{tabbed} R
 In the above example, we specified that we want to use R in our project by including a date in `runtime.txt`.
-The date tells Binder which MRAN snapshot to source R and packages from.
-[MRAN](https://mran.microsoft.com/) hosts a "CRAN Time Machine" of daily snapshots of the CRAN R packages and R releases as far back as Sept. 17, 2014.
-In the above example, the MRAN snapshot dated `r-2020-05-26` is used and the version of R and `readr` available at that date and installed.
+The date tells Binder which CRAN snapshot to source R and packages from.
+These snapshots are sourced from the [RStudio Package Manager](https://www.rstudio.com/products/package-manager/).
+In the above example, the MRAN snapshot dated `r-2021-12-20` is used and the version of R and `readr` available at that date and installed.
```

Comment:
The packagemanager.rstudio.com change currently will only be effective for dates frozen after 2022-01-01.